torch/csrc/jit/python/script_init.cpp

```diff
@@ -1756,6 +1756,7 @@ void initJitScriptBindings(PyObject* module) {
   m.def("_get_max_operator_version", &getMaxOperatorVersion);
   m.def("_get_operator_version_map", &get_operator_version_map);
   m.def("_get_upgrader_ranges", &getUpgradersRangeForOp);
+  m.def("_is_op_current", &isOpSymbolCurrent);
```

Comment:
Normally, it is slightly better for C++ function and Python method name to approximately match to reduce cognitive load when reading code.